docs/admin/vm-ops/upgrade.md

```diff
@@ -0,0 +1,37 @@
+---
+tags:
+  - administration
+  - VM
+  - operations
+  - upgrade
+---
+
+This guide shows how to upgrade a Janssen VM deployment.
+
+!!! Note
+    VM deployments don't provide automatic upgrade/update support. 
+    Uninstalling and re-installing with newer binaries is the only option that requires re-configuration of auth-server. 
+    Though this is made easy using [Terraform](../terraform/README.md). 
+    We recommend using Kubernetes installations over VM, to avail smooth upgrades and better HA support.
+
+
+Let's assume we are upgrading Jans VM installation from version `1.0.14` to `1.0.15`
```

Comment:
rephrase the assumption to remove `1.0.14` . You can simply rephrase it to `current version` to `vreplace-janssen-version`